route53.tf

```diff
@@ -31,7 +31,7 @@ resource "aws_route53_record" "hms_readonly_alias" {
 }
 
 resource "aws_route53_zone" "apiary" {
-  count = var.hms_instance_type == "k8s" ? 1 : 0
+  count = var.hms_instance_type == "k8s" && var.enable_vpc_endpoint_services ? 1 : 0
```

Comment:
Does this mean that if we don't have any endpoint services being created (because we don't have any customer_accounts defined), that we won't create the internal route53 records for HMS for the `.lcl` domain?  Why would we want to delete those?  We use those records all the time within the account itself.

Reply:
this creates additional k8s loadbalancer, nlb, security group rules, route53 entires, we don't always use these and just configure services to use waggle-dance, same is case for analytics usecase currently, this is still enabled by default.

Reply:
Just so I understand, with this change, if `enable_vpc_endpoint_services` is `false`, then the HMS service type in K8S is `ClusterIP`, which means that the HMS will only be accessible from within the K8S cluster, correct?

Reply:
correct, it will only create k8s internal service, which can be use with waggle-dance.